### ISSUE TITLE
[Suggestion] Daisy chaining bypasses

### ISSUE THREAD

**Issue by Galo223344** [Sainan/Universal-Bypass#1385]
Hello this extension is amazing and it has saved me tons of time, however it can be faster.
When people chain together several link shorteners at the same time you have to wait until the page finished loading before Universal Bypass can take you to the destination and when you have several shorteners at the same time those seconds add up. So my suggestion is that when skipping a shortener (Either using crowd bypass or other methods) Universal Bypass checks if the destination URL is in Crowd Bypass and if it finds it, it pulls that link and then it checks it to see if its in crowd bypass, this process is repeated until a link can't be skipped or the destination has been found. Here, let me draw you a little diagram 
![ ](https://user-images.githubusercontent.com/46462621/87757276-c16d7800-c7e0-11ea-9ca7-bfe4bed8586d.png)

**Comment by Sainan** [comment]
It's great that you think so well of crowd bypass but you still need the option for "was this not correct?" :)

**Comment by Galo223344** [comment]
> It's great that you think so well of crowd bypass but you still need the option for "was this not correct?" :)

In that case you can just put 2 links. One leads to the next link with the "was this not correct" dialogue and the other leads to the "Final destination".

**Comment by Sainan** [comment]
but then the semantics might be unclear to the user, and the database would need to be queried more, in what I assume to be an edge case because I don't think most shorteners even allow chaining

**Comment by Galo223344** [comment]
I find quite regularly links that contain up to five different shorteners
all chained together

How about make it an experimental feature that you have to enable and see
what's the reception?

**Comment by Sainan** [comment]
Fair enough, I'll keep this open and see if someone creates a PR, I probably won't bother
